<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsPanel</class>
 <widget class="QWidget" name="OptionsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Options for the EspINA session to be loaded.</string>
     </property>
     <property name="text">
      <string>Session Options</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="m_checkAnalysis">
     <property name="toolTip">
      <string>Check the analysis consistency for errors
after loading the file.</string>
     </property>
     <property name="text">
      <string>Check analysis for errors after loading.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 0px;
    margin-top: 1ex; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; 
}</string>
     </property>
     <property name="title">
      <string>SEG File Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="m_toolSettings">
        <property name="toolTip">
         <string>Loads all the tools parameters from the SEG file
instead of using the tool's default parameters.</string>
        </property>
        <property name="text">
         <string>Load tool settings stored in SEG file.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Options regarding the stack streaming.</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 0px;
    margin-top: 1ex; 
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center; 
}</string>
     </property>
     <property name="title">
      <string>Stack Streaming</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QRadioButton" name="m_usePreviousSettings">
        <property name="text">
         <string>Use previous settings (if loading a SEG file).</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="m_useStackStreaming">
        <property name="text">
         <string>Force use of stack streaming from disk.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
